jcl/src/java.base/share/classes/java/lang/StackTraceElement.java

```diff
@@ -243,7 +315,18 @@ public boolean isNativeMethod() {
 @Override
 public String toString() {
 	StringBuilder buf = new StringBuilder(80);
-	Util.printStackTraceElement(this, source, buf, false);
+	/**
+	 * includeExtendedInfo is essentially a check to see if this StackTraceElement
+	 * was created using a public constructor. Both includeClassLoaderName and
+	 * includeModuleVersion are initialized to true in the public constructors,
+	 * since these pieces of information are to be included in the stack trace.
+	 *
+	 * It's also possible that both of these flags are set to true natively,
+	 * in which case we still want to include extended info when printing the
+	 * stack trace.
+	 */
+	boolean includeExtendedInfo = includeClassLoaderName && includeModuleVersion;
```

Comment:
This needs to be included only for JAVA_SPEC_VERSION >= 11. If you want to change other related preprocessor statements to use JAVA_SPEC_VERSION >= 11 instead of Sidecar19-SE that's good too.